lua/starfall/libs_sh/color.lua

```diff
@@ -5,6 +5,33 @@ local dgetmeta = debug.getmetatable
 local math_Clamp = math.Clamp
 local clamp = function(v) return math_Clamp(v, 0, 255) end
 
+local string_sub = string.sub
+local hex_to_rgb = {
+	[3] = function(v) return {
+		tonumber(v[1], 16),
+		tonumber(v[2], 16),
+		tonumber(v[3], 16),
```

Comment:
You forgot to multiply by 17. Multiplying by 17 scales the values to the range of 0 to 255.

Reply:
Ah, dunno why I removed it lol